apps/windows/readme.md

```diff
@@ -25,7 +25,14 @@ As with normal UWP app development, the UWP test harness that loads the JS bundl
 ## Running the App
 
 1. Run `yarn` from the root of the repo
-2. From this directory, simply `yarn windows`
+2. From the test app folder, generate the test app from `react-native-test-app`
+
+```
+    cd apps/windows
+    yarn install-windows-test-app
```

Comment:
Could the `yarn windows` command also run `install-windows-test-app`?

Reply:
I could... I could change the following in our `package.json`
`"windows": "react-native run-windows" ` => ` "windows" : "install windows-test-app && react-native run-windows"`

Just like how I could have done this for the iOS app
`"ios": "react-native run-ios ..."` => `"ios": "pod install && react-native run-ios"`

Buuuut something felt wrong about running two commands there to me. Not sure if that's mostly unfounded 

Regardless, I could see that as a simplifying follow up change where I actually use that `fluentui-scripts` package we have to define tasks and run those.

Reply:
@tido64 any idea why the CI over here is failing? I tried doing both scripts in `yarn windows` but that failed too

Reply:
Comparing with RNTA, onedifference I'm seeing is that we are specifying `--sln` to help the CLI find the solution. But here, the app builds so I don't think that's it. The other thing we can try is to not specify `--release`. I couldn't make that work locally due to some other issue:
```
$ react-native run-windows --sln windows/Example.sln --release
 √ Auto-linking...
Success: No auto-linking changes necessary. (1404ms)
 √ Found MSBuild v16.0 at C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin (16.7.30523.141)
 √ Restoring NuGet packages
 √ Found Solution: D:\Source\react-native-test-app\example\windows\Example.sln
 i Build configuration: Release
 i Build platform: x86
 × Building Solution: D:\Source\react-native-test-app\example\node_modules\react-native-windows\PropertySheets\Bu...
 × Build failed with message      9>Failed to construct transformer : error : EBUSY: resource busy or locked, open 'D:\Source\react-native-test-app\example\msbuild.ProjectImports.zip' [D:\Source\react-native-test-app\example\node_modules\.generated\windows\ReactTestApp\ReactTestApp.vcxproj]. Check your build configuration.
```

Reply:
Okay, I found a workaround for the EBUSY error (in https://github.com/microsoft/react-native-windows-samples/pull/174) and finally reproed the issue. We are outputting a `.msix` file but `@react-native-windows/cli` expects `.appx` (submitted a fix here https://github.com/microsoft/react-native-windows/pull/6421). This happens if `WindowsTargetPlatformMinVersion` is high enough. Patching it locally, I can get passed the error but I'm hitting some issues with app signing. I can look into it when I have some time later...

Reply:
You will also need this patch: https://github.com/microsoft/react-native-test-app/pull/241. You'll need both this and the `@react-native-windows/cli` patch. I don't know if you can bump `@react-native-windows/cli` alone. If not, you'll have to look into patching it somehow.

Reply:
I can probably wait till both `react-native-test-app` and `react-native-windows` bump, and update the dependencies and rerun the CI. In the meantime, I'll try to apply changes locally and see if I can find more issues

Reply:
FYI, both PRs are merged. For the fixes in `react-native-test-app`, you should use 0.2.16. The fix in `@react-native-windows/cli` is not yet released. It will likely hit 0.64 if I understand the release process correctly.